kie-api/src/main/java/org/kie/api/internal/assembler/KieAssemblerService.java

```diff
@@ -15,13 +15,24 @@
 
 package org.kie.api.internal.assembler;
 
+import java.util.Collection;
+
 import org.kie.api.internal.utils.KieService;
 import org.kie.api.io.Resource;
 import org.kie.api.io.ResourceConfiguration;
 import org.kie.api.io.ResourceType;
+import org.kie.api.io.ResourceWithConfiguration;
 
 public interface KieAssemblerService extends KieService {
     ResourceType getResourceType();
 
+    default void addResources(Object kbuilder, Collection<ResourceWithConfiguration> resources, ResourceType type) throws Exception {
```

Comment:
I assume there is no suitable interface/class that could be used here instead of Object?

Reply:
@winklerm correct, and this is for consistency with the other method in class `KieAssemblerService`